src/bp/admin/ui/config-overrides.js

```diff
@@ -2,77 +2,110 @@ const path = require('path')
 const ModuleScopePlugin = require('react-dev-utils/ModuleScopePlugin')
 const HardSourceWebpackPlugin = require('hard-source-webpack-plugin')
 const isProductionBuild = process.argv.includes('--prod')
+const { createProxyMiddleware } = require('http-proxy-middleware')
 
-module.exports = function override(config, env) {
-  config.resolve.alias['common'] = path.join(__dirname, '../../../../out/bp/common')
-  config.resolve.alias['~'] = path.join(__dirname, './src')
-  config.resolve.alias['botpress/shared'] = 'ui-shared'
-  config.resolve.plugins = config.resolve.plugins.filter(p => !p instanceof ModuleScopePlugin)
-  config.devtool = process.argv.find(x => x.toLowerCase() === '--nomap') ? false : 'source-map'
+module.exports = {
+  webpack: (config, env) => {
+    config.resolve.alias['common'] = path.join(__dirname, '../../../../out/bp/common')
+    config.resolve.alias['~'] = path.join(__dirname, './src')
```

Comment:
ca we get rid of this ~ notation ? or can we use it everywhere. It's a bit confusing to have it working sometimes and sometimes not don't you think

Reply:
I'd suggest we use it everywhere then, paths are clearer and when moving files around they are correctly updated, while ../.. sometimes are not updated correctly